.github/workflows/node-flow-pull-request-checks.yaml

```diff
@@ -255,7 +255,7 @@ jobs:
     needs:
       - dependency-check
       - spotless
-    if: ${{ github.event_name != 'pull_request' || (github.actor != 'dependabot[bot]' && github.event.pull_request.head.repo.full_name == github.repository) }}
```

Comment:
We need to leave the JRS panel disabled for dependabot per conversations with @nathanklick